modules/learn/pages/security/roles.adoc

```diff
@@ -1097,6 +1097,191 @@ The user can, by means of Sync Gateway, read and write data, manage indexes and
 ^| image:introduction/no.png[]
 |===
 
+[#sync-gateway-architect]
+== Sync Gateway Architect
+
+The *Sync Gateway Architect* role (which is a _Mobile_ role) allows management of Sync Gateway databases; and of Sync Gateway users, groups, and roles; and allows access to Sync Gateway's `/metrics` endpoint.
+The role does not allow access to Couchbase Web Console; and does not allow reading of application data.
+
+[#table_sync_gateway_architect_role,cols="15,8,8,8,8",hrows=3]
+|===
+5+^| Role: Sync Gateway Architect (`mobile_sync_gateway_architect`)
+
+.2+^h| Resources
+4+^h| Privileges
+
+^h| *Read*
+^h| *Write*
+^h| *Execute*
+^h| *Manage*
+
+^| UI
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+
+^| Collection: Data
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+
+^| Collection: Sync Gateway Users, Groups, and Roles
+^| image:introduction/yes.png[]
+^| image:introduction/yes.png[]
+^| image:introduction/yes.png[]
+^| image:introduction/yes.png[]
+
+^| Metrics: Sync Gateway
+^| image:introduction/yes.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+^| image:introduction/no.png[]
+|===
+
+[#sync-gateway-application]
+== Sync Gateway Application
+
+The *Sync Gateway Application* role (which is a _Mobile_ role) allows management of Sync Gateway users, groups, and roles; and allows application data to be read and written through Sync Gateway.
+The role does not allow access to Couchbase Web Console.
```

Comment:
@tonyjhillman  Have to check on this one (for all roles). Not quite sure if our implementation of roles enforces this